src/Sulu/Bundle/AdminBundle/Command/DownloadBuildCommand.php

```diff
@@ -147,22 +165,118 @@ protected function execute(InputInterface $input, OutputInterface $output)
         \unlink($tempFileZip);
     }
 
-    private function getLocaleFileHash(string $path)
+    private function getLocalFile(string $path)
     {
-        return $this->hash(\file_get_contents($this->projectDir . $path));
+        if (!\file_exists($this->projectDir . $path)) {
+            return '';
+        }
+
+        return \file_get_contents($this->projectDir . $path);
     }
 
-    private function getRemoteFileHash(string $path)
+    private function getRemoteFile(string $path)
     {
         $path = \str_replace(\DIRECTORY_SEPARATOR, '/', $path);
         $response = $this->httpClient->request('GET', $this->remoteRepository . $path);
 
-        return $this->hash($response->getContent());
+        return $response->getContent();
     }
 
     private function hash($content)
     {
         // we remove all whitespaces as the developer could change the indention or/and the line breaks of this files
         return \hash('sha256', \preg_replace('/\s+/', '', $content));
     }
+
+    private function writeFile(string $path, string $content): void
+    {
+        \file_put_contents($this->projectDir . $path, $content);
+    }
+
+    private function doManualBuild(SymfonyStyle $ui): int
+    {
+        $ui->warning(\sprintf(
+            'The files in the local "%s" folder do not match the ones in the remote repository "%s".' . \PHP_EOL
```

Comment:
`The files in the local "%s" folder do not match the ones in the remote repository "%s". If you have added custom JavaScript to the administration interface, you need to create the JavaScript build by yourself. Do you want to create a build now?`